.github/workflows/push.yaml

```diff
@@ -0,0 +1,136 @@
+name: Test Suite
+
+on:
+  push:
+    branches:    
+      - '**'
+
+jobs:
+  linter:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name:
+        run: |
+          #!/bin/bash
+          set -euxo pipefail
+          python3 -m pip install -q flake8==3.8.4
```

Comment:
Currently `flake8` is in version 6.0.0 (<https://pypi.org/project/flake8/#history>). Does any test fail with the most recent version?